docs/1.x/themes/change-email-template.md

```diff
@@ -0,0 +1,98 @@
+# Change email template
+
+In this section, we are explaining how to change the email template in the Bagisto. As we all know that everyone wants some customization according to their own taste that's why knowing about changing the email template is also quite helpful for some users.
```

Comment:
In this section, we are explaining how to change the email templates in Bagisto. As we all know, everyone wants some customization according to their own taste that's why knowing about changing the email template is also quite helpful for some users.

Reply:
Done!